controller/planner_backlog.go

```diff
@@ -59,9 +59,18 @@ func (c *PlannerBacklogController) List(ctx *app.ListPlannerBacklogContext) erro
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, err)
 	}
+	// Load all work item types
+	wits := make([]workitem.WorkItemType, len(result))
+	for idx, wi := range result {
+		wit, err := c.db.WorkItemTypes().Load(ctx.Context, wi.Type)
```

Comment:
This DB operation call is not wrapped under `application.Transactional`. Is that intentional as it's a read operation?

Reply:
@baijum it is fine access it this way. And there is no `application.Transactional` anywhere in this function, is it? I don't see a point in spawning a transaction just to query for types.

Reply:
@baijum but I think I have written a function that does the same as this. Let me rework this bit.

Reply:
@baijum Please see 9c03db13

Reply:
I am fine that the read operation is not wrapped under a transaction. Though, in line number 58 there is a call to `getBacklogItems` which actually creates a transaction wrapper.

Reply:
@baijum you're right. In 1f2d7fad I've move the loading of WITs to `getBacklogItems`. Thanks for pointing that out.